CodeChallenge/Challenges/LetterCombinationsOfPhoneNumber/Entries/juliand665LetterCombinationOfPhoneNumberEntry.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  juliand665LetterCombinationOfPhoneNumberEntry.swift
+//  CodeChallenge
+//
+//  Created by Julian Dunskus on 06.01.17.
+//  Copyright © 2017 iosdevelopers. All rights reserved.
+//
+
+import Foundation
+
+let juliand665LetterCombinationOfPhoneNumberEntry = CodeChallengeEntry<LetterCombinationsOfPhoneNumberChallenge>(name: "juliand665") { input in
+	
+	var combs: [String] = [""]
+	
+	let expanded = input.characters.flatMap { numbersToLetters[$0] }
+	
+	for letters in expanded {
+		combs = combs.flatMap { prev in
+			letters.map { prev.appending($0) }
+		}
+	}
+	
+	return combs
+}
+
+private let numbersToLetters: [Character: [String]] = [ // explicit declaration reduces compile time
```

Comment:
I need to update the README for Swift 3, but this should be `fileprivate`. Don't worry about changing it @juliand665, I'll do it when I update the README, but just wanted to point it out here.